test/datasets.js

```diff
@@ -399,6 +376,109 @@ test('DatasetClient', function(datasetClient) {
     bulkFeatureUpdate.end();
   });
 
+  datasetClient.test('#listFeatures', function(listFeatures) {
+    var featureIds = [];
+
+    // Setup data
+    listFeatures.test('insert some', function(assert) {
+      var client = new MapboxClient(process.env.MapboxAccessToken);
+      assert.ok(client, 'created dataset client');
+      var features = randomFeatures(3).map(function(f) {
+        f.id = hat();
```

Comment:
@rclark the line below though I am pushing it into a `featuresIds` container to keep track. But agree, this could just be more simple.